pkg/network/payload/version.go

```diff
@@ -19,7 +19,7 @@ type Version struct {
 	Version uint32
 	// timestamp
 	Timestamp uint32
-	// it's used to distinguish the node from public IP
+	// it's used to distinguish the node from a public IP
```

Comment:
It's not written well originally, I guess. It allows to distinguish several nodes using the same IP (or different ones also).

Reply:
Yep, looks good enough.